Mathlib/Topology/Homotopy/Product.lean

```diff
@@ -120,7 +120,7 @@ namespace Path.Homotopic
 
 attribute [local instance] Path.Homotopic.setoid
 
-local infixl:70 " ⬝ " => Quotient.comp
+local infixl:70 " * " => Quotient.comp
```

Comment:
Is this desired?

Reply:
Nope, this was completely accidental, good catch